### DISCUSSION TITLE
Cannot change program version number when using `wingetcreate update`

### DISCUSSION THREAD

**Discussion by Joey451-OG** [microsoft/winget-create#366] [Q&A] [answered]
I'm trying to update my manifest because I released a new version, but when I use `wingetcreate update -i` I cannot change my program's version number (1.2.1 --> 1.3.0). I can edit the ymal files to replace the download url and version number manually, but how can I submit that edited manifest as an update. Sorry if I'm not seeing something obvious I'm kinda hitting my head against a wall.

**Comment by mdanish-kh** [comment, marked as answer]
For an interactive update to a newer version, you can use `wingetcreate update <PackageIdentifier> --version <NewVersion> -i`

**Reply by mdanish-kh** [reply]
Just an fyi: You can also have it run unattended with
`wingetcreate update <PackageIdentifier> -v <NewVersion> -u <InstallerUrl>|<Architecture> <InstallerUrl2>|<Architecture>...` and automatically submit to winget-pkgs if you also add a `--submit` flag

**Reply by Joey451-OG** [reply]
Thanks, that worked! For some reason I had been using `--version` wrong....
